bin/plugin/commands/readme.js

```diff
@@ -0,0 +1,177 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const fs = require( 'fs' );
+
+/**
+ * Internal dependencies
+ */
+const { log, formats } = require( '../lib/logger' );
+const config = require( '../config' );
+const { getModuleDescriptions } = require( './common' );
+const { getChangelog } = require( './changelog' );
+
+/**
+ * @typedef WPReadmeCommandOptions
+ *
+ * @property {string=} directory Optional directory, default is the root `/modules` directory.
+ * @property {string=} milestone Optional milestone title, to update the changelog in the readme.
+ * @property {string=} token     Optional personal GitHub access token, only relevant for changelog updates.
+ */
+
```

Comment:
Maybe missing the object defined above?

Reply:
Sure, I see a definition for an object `WPReadmeCommandOptions` and properties in this case `milestone` and `token` but I don't see the object itself in the doc, is this maybe document as part of the command ?